converters/fromZigbee.js

```diff
@@ -3480,14 +3484,84 @@ const converters = {
         cluster: 'hvacThermostat',
         type: ['readResponse'],
         convert: (model, msg, publish, options, meta) => {
-            const lookup = {0: 'unoccupied', 1: 'occupied'};
-            const lookup1 = {0: 'on_demand', 1: 'sensing'};
             const result = {};
+            const cycleOutputLookup = {15: '15 sec', 300: '5 min', 600: '10 min',
```

Comment:
Can you make this snake case, e.g. `15 sec` -> `15_sec`

Reply:
Done